plugins/openshift-developer/evals/scripts/run-solve.sh

```
@@ -0,0 +1,320 @@
+set -e
+
+# Diff coverage
+CHANGED_PKGS=$( { git diff "$BASE_SHA"..HEAD --name-only -- '*.go' 2>/dev/null; git diff HEAD --name-only -- '*.go' 2>/dev/null; } \
+  | grep -v '_test.go$' \
+  | xargs -I{} dirname {} 2>/dev/null \
+  | sort -u \
+  | sed 's|^|./|' || echo "")
+if [ -n "$CHANGED_PKGS" ]; then
+  echo "Running diff coverage for changed packages..."
+  set +e
+  go test -coverprofile="$OUTPUT_DIR/cover.out" $CHANGED_PKGS 2>&1 | tee "$OUTPUT_DIR/coverage-raw.txt"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)